--- a/scripts/download.pl
+++ b/scripts/download.pl
@@ -292,10 +292,10 @@ foreach my $mirror (@ARGV) {
 			push @extra, "$extra[0]/longterm/v$1";
 		}
 		foreach my $dir (@extra) {
+			push @mirrors, "https://cdn.kernel.org/pub/$dir";
 			push @mirrors, "https://mirror.nju.edu.cn/kernel.org/$dir";
 			push @mirrors, "https://mirrors.hust.edu.cn/kernel.org/$dir";
 			push @mirrors, "https://mirrors.ustc.edu.cn/kernel.org/$dir";
-			push @mirrors, "https://cdn.kernel.org/pub/$dir";
 			push @mirrors, "https://mirrors.mit.edu/kernel/$dir";
 			push @mirrors, "http://ftp.nara.wide.ad.jp/pub/kernel.org/$dir";
 		}
